docker-compose.yml

```diff
@@ -5,25 +5,27 @@ services:
     image: inputoutput/cardano-node:1.20.0
     restart: on-failure
     environment:
-      NETWORK:
+      - NETWORK=${NETWORK:-mainnet}
     volumes:
-      - node-${NETWORK}-db:/data
+      - node-db:/data
       - node-ipc:/ipc
     logging:
-      driver: none
+      driver: "json-file"
+      options:
+        max-size: "400k"
+        max-file: "20"
 
   ogmios:
     image: ktorz/ogmios:latest
     restart: on-failure
     environment:
-      - OGMIOS_NETWORK=${NETWORK}
+      - OGMIOS_NETWORK=${NETWORK:-mainnet}
     command: ["--host", "0.0.0.0", "--node-socket", "/ipc/node.socket"]
     volumes:
       - node-ipc:/ipc
     ports:
-      - 1337:1337
+      - ${OGMIOS_WS_PORT:-1337}:1337
```

Comment:
Note that this port is used for both the websocket and also "classic" http requests (like `/health`, `/benchmark.html` or `/ogmios.wsp.json`). I'd rather call that one simply `OGMIOS_PORT` :+1:

Reply:
Will generalise